src/System.Runtime/tests/System/Runtime/Tests1.7.cs

```diff
@@ -0,0 +1,33 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Should we change the name of the file here to be something like: RuntimeHelpersTests.netstandard1.7.cs?